composer.json

```diff
@@ -2,7 +2,7 @@
   "name": "adyen/module-payment",
   "description": "Official Magento2 Plugin to connect to Payment Service Provider Adyen.",
   "type": "magento2-module",
-  "version": "2.4.3",
+  "version": "2.4.4",
```

Comment:
we should go for the 3.0.0 version since we removed the POS and made some breaking changes

Reply:
I would not go for 3.0.0 for pos removal. Let's discuss versioning for the release. This needs to be updated because of db changes.